### PR TITLE
Rename to Abstractions.io

### PR DESCRIPTION
**Pull request by pfaffle**
Rename all the string references to BABBQ to either nothing or Abstractions.io. Update the conference venue information and Google Maps hyperlink. Update the authors and remove the link to the original app author's Twitter. Update README. Update the app ID.

@jasonaowen

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -13,11 +13,12 @@
     <string name="day3">Saturday</string>
     <!-- about -->
     <string name="venue">Venue</string>
-    <string name="venue_text">Big Android BBQ will take place at B.Amsterdam\n\nMetro line 50 Gein / Isolatorweg – get off at stop Henk Sneevlietweg</string>
+    <string name="venue_map_url">https://www.google.com/maps/place/David+L.+Lawrence+Convention+Center/@40.4454067,-80.0052605,15z/data=!3m1!5s0x8834f3e2e996afd9:0x3bff6f9c14520247!4m12!1m6!3m5!1s0x0:0x36ba9693af60a61!2sDavid+L.+Lawrence+Convention+Center!8m2!3d40.4457986!4d-79.9961624!3m4!1s0x0:0x36ba9693af60a61!8m2!3d40.4457986!4d-79.9961624</string>
+    <string name="venue_text">Abstractions.io 2016 will take place at The David L. Lawrence Convention Center in Pittsburgh, PA.</string>
     <string name="opensource">Open Source</string>
     <string name="opensource_text">List of tools and libraries used to make this app.</string>
-    <string name="author">Author</string>
-    <string name="author_text">This app has been developed by Arnaud Camus.</string>
```
Comment (jasonaowen) [inline app/src/main/res/values/strings.xml:20]: We should add an entry to the open source library list to give credit to the original author.

**Comment by jasonaowen** [comment]
:+1: :ship: :it:

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -38,6 +38,15 @@
     <string name="notification_text">We hope you can help us improve the event by providing some feedback information</string>
 
     <!-- libraries -->
+    <string name="define_BABBQ"></string>
+    <string name="library_BABBQ_author">Arnaud Camus</string>
+    <string name="library_BABBQ_libraryName">Big Android BBQ 2015 App</string>
+    <string name="library_BABBQ_libraryDescription">Simple conference app made for various conferences. Displays the list of talks and details about them: speaker, description, room, etc.</string>
+    <string name="library_BABBQ_libraryVersion">1.0.1</string>
```
Comment (pfaffle) [inline app/src/main/res/values/strings.xml]: Didn't we just bump it to 1.0.1?

**Comment by pfaffle** [comment]
:ship: it

**Pull request merged**